xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -409,6 +409,7 @@ namespace PVR
     CDateTime m_recordingTime; /*!< start time of the recording */
     bool m_bGotMetaData;
     bool m_bIsDeleted; /*!< set if entry is a deleted recording which can be undelete */
+    mutable bool m_bInProgress; /*!< set if recording might be InProgress. */
```

Comment:
"might be in progress", not "might be InProgress", please.

Reply:
commit and squashed